code/iaas/service-discovery/api/src/main/java/io/cattle/platform/servicediscovery/api/resource/ServiceDiscoveryConfigItem.java

```diff
@@ -58,6 +58,9 @@
 
     public static final ServiceDiscoveryConfigItem LABELS = new ServiceDiscoveryConfigItem(
             InstanceConstants.FIELD_LABELS, InstanceConstants.FIELD_LABELS);
+    public static final ServiceDiscoveryConfigItem MEMSWAPLIMIT = new ServiceDiscoveryConfigItem("memorySwap", "memswap_limit");
```

Comment:
Can you change this param to be read from DockerInstanceConstants.java? And the params below should also be read from constants. No need to convert to "_", underlying code will do it for you.